lms/djangoapps/certificates/docs/decisions/001-allowlist-cert-requirements.rst

```diff
@@ -31,7 +31,9 @@ the time the certificate is generated:
   * The enrollment mode must be eligible for a certificate
   * The enrollment does not need to be active
 
-* The user must have an approved, unexpired, ID verification
+* The user must be verified through a `Name Affirmation service`_
```

Comment:
Is Name Affirmation the only option now? Can we decide to use verified ids as a fallback still? This is one thing I was confused about when we introduced Name Affirmation.